trainer/train_util.py

```diff
@@ -0,0 +1,204 @@
+# Copyright 2020 The DDSP Authors.
```

Comment:
I guess this file is not from the Google style pylint, as I can see 2 spaces and 2 blank lines between functions. In this situation when you want to take files from different sources and don't want to change them to use our formatting rules, you should keep them in a separate directory with a clear readme pointing to the origin.

Reply:
Okay, I will restructure it a little and move all internal Magenta files to the separate folder inside the trainer directory.